components/org.wso2.carbon.identity.api.server.dcr/src/gen/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/dto/RegistrationRequestDTO.java

```diff
@@ -40,14 +41,28 @@ public class RegistrationRequestDTO  {
   private boolean extPkceMandatory;
   private boolean extPkceSupportPlain;
   private boolean extPublicClient;
-  
+  private String tokenEndpointAuthMethod = null;
+  private String tokenEndpointAuthSigningAlg = null;
+  private String sectorIdentifierUri = null;
+  private String idTokenSignedResponseAlg = null;
+  private String idTokenEncryptedResponseAlg = null;
+  private String idTokenEncryptedResponseEnc = null;
+  private String authorizationSignedResponseAlg = null;
```

Comment:
With the last discussion, we can remove this right?

Reply:
removed